cmd/srcd/cmd/init_test.go

```diff
@@ -0,0 +1,205 @@
+// +build integration
+
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	cmdtest "github.com/src-d/engine/cmd/test-utils"
+	"github.com/src-d/engine/components"
+	"github.com/stretchr/testify/suite"
+)
+
+type InitTestSuite struct {
+	cmdtest.IntegrationSuite
+	timeout        time.Duration
+	testDir        string
+	validWorkDir   string
+	invalidWorkDir string
+}
+
+func TestInitTestSuite(t *testing.T) {
+	itt := InitTestSuite{timeout: 1 * time.Minute}
+	suite.Run(t, &itt)
+}
+
+// 1. Build docker image for daemon and name it like dev-blabla-dirty (make it automatic)
+// 2. make build
+// 3.
```

Comment:
should something be here?

Reply:
This was a personal note 😂. Gonna remove it.